### PR TITLE
created new  class

### PR DESCRIPTION
**Pull request by KeerthiPriya111**
Please review

### CODE REVIEW

File: Project1/src/devisamplepro/Dev_iclass2.java
Comment (KeerthiPriya1116) [inline Project1/src/devisamplepro/Dev_iclass2.java:1]: Do not hardcode data

**Review by KeerthiPriya1116 — COMMENTED** [review]
added comments

**Comment by KeerthiPriya111** [comment]
fixed

**Comment by KeerthiPriya1116** [comment]
merged

**Pull request merged**